### ISSUE TITLE
new mode that control each motor separately

### ISSUE THREAD

**Issue by Nitzanbendavid** [betaflight/betaflight-configurator#2036]
Hi everyone,
I would like to add a new mode in my quad where I can control each motor separately, without the intervention of the flight controller. The reason is, that I want my quad also to drive on walls, and for this I need to have the option than only two motors will spin the other direction, while the other two motors are connected to the wheels and in charge of the steering and driving. (it means that two motors making thrust in the opposite direction in order to stick the quad to the wall, and the other two motors are driving the wheels.) Tottaly there are 4 motors.
The bottom line, how can I add a new mode, where I can control on each motor separate?

Thanks for helping,
Nitzan

**Comment by mikeller** [comment]
@Nitzanbendavid: This is the wrong place to ask for new features in the Betaflight firmware. Also, please don't crosspost across multiple projects - this is rude. Please close this issue.
